GSA-TTS/identity-site
# LG-13909 | Remove some verify-by-mail references

## 🎫 Ticket

Link to the relevant ticket.
[LG-13909](https://cm-jira.usa.gov/browse/LG-13909)

## 🛠 Summary of changes
We may need to disable GPO during biometric proofing (for compliance reasons). Update the relevant help center articles now so they're not misleading/frustrating.


## 📸 Previews
### How to verify your identity
* [English](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/gsa-tts/identity-site/mattw/LG-13909/help/verify-your-identity/how-to-verify-your-identity/)
* [Chinese](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/gsa-tts/identity-site/mattw/LG-13909/zh/help/verify-your-identity/how-to-verify-your-identity/)
* [French](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/gsa-tts/identity-site/mattw/LG-13909/fr/help/verify-your-identity/how-to-verify-your-identity/)
* [Spanish](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/gsa-tts/identity-site/mattw/LG-13909/es/help/verify-your-identity/how-to-verify-your-identity/)

### Verify by mail
* [English](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/gsa-tts/identity-site/mattw/LG-13909/help/verify-your-identity/verify-your-address-by-mail/)
* [Chinese](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/gsa-tts/identity-site/mattw/LG-13909/zh/help/verify-your-identity/verify-your-address-by-mail/)
* [French](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/gsa-tts/identity-site/mattw/LG-13909/fr/help/verify-your-identity/verify-your-address-by-mail/)
* [Spanish](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/gsa-tts/identity-site/mattw/LG-13909/es/help/verify-your-identity/verify-your-address-by-mail/)
